Footer reorganization

trwnh opened the pull request:
- Clicking pixelfed.org should load pixelfed.org
- Order of footer should more closely match site routes menu

trwnh committed:
Link footer to pixelfed.org

Clicking pixelfed.org should load pixelfed.org

trwnh committed:
Reorder footer

Organization more closely matches site routes menu

hellcp commented:
I would change it to "Powered by PixelFed", it would be nice towards later instances to not brand them as belonging to "PixelFed.org", instead just create indicator that it's powered by this project ;)

trwnh commented:
Good idea

trwnh committed:
"Powered by PixelFed"

Copyright status isn't really appropriate since PF isn't a commercial project and can be self-hosted across many instances
